Incorrect IAmqpConnectionManager interface declaration

Can't build TS project uses `amqp-connection-manager` because of errors:
```
node_modules/amqp-connection-manager/dist/esm/AmqpConnectionManager.d.ts(103,5): error TS1131: Property or signature expected.
node_modules/amqp-connection-manager/dist/esm/AmqpConnectionManager.d.ts(103,9): error TS1005: ';' expected.
node_modules/amqp-connection-manager/dist/esm/AmqpConnectionManager.d.ts(103,21): error TS1005: ';' expected.
node_modules/amqp-connection-manager/dist/esm/AmqpConnectionManager.d.ts(105,9): error TS1005: ';' expected.
node_modules/amqp-connection-manager/dist/esm/AmqpConnectionManager.d.ts(105,23): error TS1005: ';' expected.
node_modules/amqp-connection-manager/dist/esm/AmqpConnectionManager.d.ts(106,1): error TS1128: Declaration or statement expected.
```

This issue can be resolved by replacing [these lines](https://github.com/jwalton/node-amqp-connection-manager/blob/master/src/AmqpConnectionManager.ts#L115-L119)
```ts
    /** The current connection. */
    get connection(): Connection | undefined;

    /** Returns the number of registered channels. */
    get channelCount(): number;
```

with proper "getter-only" definition
```ts
    /** The current connection. */
    readonly connection: Connection | undefined;

    /** Returns the number of registered channels. */
    readonly channelCount: number;
```

--- a/src/AmqpConnectionManager.ts
+++ b/src/AmqpConnectionManager.ts
@@ -113,10 +113,10 @@ export interface IAmqpConnectionManager {
     isConnected(): boolean;
 
     /** The current connection. */
-    get connection(): Connection | undefined;
+    readonly connection: Connection | undefined;
 
     /** Returns the number of registered channels. */
-    get channelCount(): number;
+    readonly channelCount: number;
 }
 
 //

